toqito/rand/random_circulant_gram.py

```diff
@@ -0,0 +1,45 @@
+"""Generate random circulant Gram matrix."""
+import numpy as np
+
+
+def random_circulant_gram(dim: int) -> np.ndarray:
+    r"""Generate a random circulant Gram matrix of specified dimension.
+
+    A circulant matrix is a square matrix where each row is a shifted version of the one above,
```

Comment:
Something seems a bit off with the formatting of these lines. They should have a length of 120, but these look like they are around 80.

Reply:
Probably because `black` was used for formatting issues?

Reply:
yes, it was because i was using formatter in wrong way. i have corrected it and do let me know if any other further changes is required.